### ISSUE TITLE
Please add option to check multiple pincodes

### ISSUE THREAD

**Issue by hpk22** [1itromio/cowin-vaccine-booking#3]

**Comment by ManojMadanmohan** [comment]
It looks like this is unsupported by the API: https://apisetu.gov.in/public/api/cowin#/Appointment%20Availability%20APIs/calendarByPin

not sure if this would be doable

**Comment by 1itromio** [comment]
Yeah it does not support that, However we can save the pincodes in local and try for each alternatively. However this will slow down the polling interval for one pincode considering the rate limit of cowin APIs applicable to individual IP Address. Rather we can use a strategy to switch between mobile network and wifi interface to make the api calls so that the IP addresses differ for api calls and we get a less chance of getting 301
